src/alert_manager.cpp

```diff
@@ -75,7 +74,7 @@ namespace libtorrent
 	{
 		if (a->type() == save_resume_data_failed_alert::alert_type
 			|| a->type() == save_resume_data_alert::alert_type)
-			++m_num_queued_resume;
+			m_num_queued_resume.fetch_add(1, boost::memory_order_relaxed);
```

Comment:
I don't understand why you would use relaxed here. This counter is definitely tied to the actual alerts stored in the queue, it's not independent. This counter better be consistent with the queue itself, by making it relaxed it won't synchronise *any* other memory. Again, I think anything other than "sequential consistency" requires strong arguments and good rationale.